src/clevertap.js

```diff
@@ -209,6 +287,66 @@ export default class CleverTap {
 
     this.setLogLevel = (l) => {
       this.#logger.logLevel = Number(l)
+      if (l === 3) {
+        sessionStorage.WZRK_D = ''
+      } else {
+        delete sessionStorage.WZRK_D
+      }
+    }
+    /**
+ * @param {} key
+ * @param {*} value
+ */
+    this.handleIncrementValue = (key, value) => {
+      this.profile._handleIncrementDecrementValue(key, value, COMMAND_INCREMENT)
+    }
+
+    this.handleDecrementValue = (key, value) => {
+      this.profile._handleIncrementDecrementValue(key, value, COMMAND_DECREMENT)
+    }
+
+    this.setMultiValuesForKey = (key, value) => {
+      if (Array.isArray(value)) {
+        this.profile._handleMultiValueSet(key, value, COMMAND_SET)
+      } else {
+        console.log('Value should be of type array.')
```

Comment:
optional: the error message can be more descriptive like "setMultiValuesForKey should be called with a value of type array". and maybe console.error() to make it more prominent red.

Reply:
Error message corrected for all methods